docs/examples/good_practices/slurm_job_arrays/job.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#SBATCH --gpus-per-task=rtx8000:1
+#SBATCH --cpus-per-task=4
+#SBATCH --ntasks-per-node=1
+#SBATCH --mem=16G
+#SBATCH --time=00:15:00
+
+# No need to change this file, as environment variables
```

Comment:
Est-ce que ça c'est dans le single-gpu example?

Reply:
Non ! J'ai rajouté ces commentaires pour préciser qu'il n'est pas nécessaire de modifier `job.sh` pour cet exemple !

Et aussi, quand le `job.sh` n'a aucune différence, j'ai des erreurs avec le `pre-commit`, il refuse de générer les diffs. Je ne sais pas encore pourquoi, du coup, j'ai ajouté ce commentaire pour contourner le problème (pour qu'il y ait au moins une différence entre les deux fichies).

Reply:
Hmm si il n'y a aucune différence, alors pas besoin d'inclure ce fichier dans l'exemple (et donc pas besoin de créer de diff non plus)

Reply:
Ok ! Fait ! https://github.com/mila-iqia/mila-docs/pull/225/commits/3e9a5ce922debe9d2400cf30f983711ca6651e0d

Reply:
Small comment here. If a user needs the `job.sh` to run the example it should still be found in the directory right?